tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/QueryParameterSpecificRule.java

```diff
@@ -1,18 +1,17 @@
 package org.dcsa.conformance.standards.tnt.party;
 
+import static org.dcsa.conformance.standards.tnt.checks.TntDataSets.VALID_DOCUMENT_TYPE_CODES;
```

Comment:
the guidelines  suggest pacakge imports first and then the static imports.

Reply:
I suggest that we keep following the way our IDE formats the imports nowadays and that we do not manually interfere. We're not reading it nor updating it manually, so it should not matter that much. 
If your IDE reorders the imports differently than mine, we should update the `.editorconfig` file, so it is kept consistent throughout all dev environments. I expected that it was already the case. 
Did you also install the `google-java-format` plugin AND did you enable it in our project @preetamnpr ?

Reply:
Since I don't see you online and we have the sprint planning soon, I am proceeding with this PR as it is now. If still would like to talk about this topic and we later can also apply changes still. So, please reply when you have time.